astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -137,3 +137,7 @@ def make_transform_graph_docs(transform_graph):
 
 
 _transform_graph_docs = make_transform_graph_docs(frame_transform_graph)
+
+# Here, we override the module docstring so that sphinx renders the transform
+# graph without the developer documentation in the main docstring above.
+__doc__ = _transform_graph_docs
```

Comment:
Seems fine as a work-around, though even nicer would be if somehow we can include it directly in the `.rst` file...

Reply:
I'm not sure if that's possible with the current implementation, as it is generated on-the-fly. Maybe @eteq has some thoughts?

Reply:
I looked around a bit more and it does seem possible to let sphinx display just the docstring of an item: https://stackoverflow.com/questions/7825263/including-docstring-in-sphinx-documentation

But perhaps this is better done separately. cc also @astrofrog, who seems to know this kind of stuff.

Reply:
But it's not the docstring of an item, it's a docstring *generated* by a function, right? Can we do this in a separate PR since this one works with the existing machinery?

Reply:
Yes, in a separate PR may be best.

Reply:
I agree with @adrn that the main problem is that it's generally difficult to do this kind of thing during the sphinx build process because it requires code execution.  Options include:
1. Doing it in the `conf.py` or a small custom sphinx extension
2. Using an otherwise *empty* module called `.../builtin_frames/transform_graph_docs.py` that just does:
```
from astropy.coordinates.builtin_frames import frame_transform_graph
__doc__ = make_transform_graph_docs(frame_transform_graph)
```
and use `.. module:: transform_graph_docs` to import it.  But I'm not sure that's really much better than the current solution.

Thiat said, I agree this is better dealt with separately.

Reply:
I actually like the second solution! It helps to keep module `__doc__` free for development stuff (which `builtin_frames` has already; see #8930)